app/services/place_hold_form/builder.rb

```diff
@@ -11,9 +11,16 @@ def initialize(catkey:, user_token:, client:)
 
   def generate
     bib_info = Bib.new(SymphonyClientParser::parsed_response(@client, :get_bib_info, @catkey, @user_token))
+
+    # record with empty callList
+    return if bib_info.call_list.blank?
```

Comment:
Can you tell me why there wouldn't be a callList? I am trying to figure out when that scenario occurs. I just checked localhost:3001/holds/new?catkey=31200756 and even that OA electronic resource has a `callList`... 🤔

Reply:
we talked about this on zoom but just to note here: had to check the callList here because for requests that returns an error, Bib still had a @record of an error

Reply:
yeah 😒